.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,75 @@
+name: 🐞 Report a bug
+description: Create a bug report about unexpected behavior.
+labels: [kind/bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Please only use this form to submit problems with building Kraftkit or it's interaction with Unikraft unikernels.  This includes unexpected or undefined behavior, stacktraces, critical messages, and things just breaking or exploding.
+        
+        **Ensure you have tested against the latest changes on `staging` branch before submitting this form.**
+
+        For questions, concerns or help with running Kraftkit, please check out our additional resources:
+
+        * [Unikraft Documentation](https://unikraft.org/docs/)
+        * [Unikraft Discussions Forum](https://github.com/unikraft/kraftkit/discussions)
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: Please provide a clear and concise description of the bug.
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Please provide the steps necessary to reproduce the behavior, for example the command used to run Kraftkit, or the sequence of actions, auxiliary components, networking setup, etc. leading up to the error or bug.  Please also include any `kraft.yaml` files you used.
+      placeholder: |
+        For example, I used this kraft configuration:
+
+         ```
+         specification: "0.5"
+         name: helloworld
+         outdir:
+         ...
+         ```
+         - ...
+    validations:
+      required: false
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+      placeholder: For example, I expected the unikernel to boot and print "Hello, world!"
+    validations:
+      required: false
+  - type: dropdown
+    id: architectures
+    attributes:
+      label: Which architectures were you using or does this bug affect?
+      description: If this bug is architecture agnostic or you are unsure then please do not select an architecture listed below.
+      multiple: true
+      options:
+        - x86_64
+        - arm
+        - arm64
+  - type: dropdown
+    id: platforms
+    attributes:
+      label: Which operating system were you using or does this bug affect?
+      description: If this bug is operating system agnostic or you are unsure then please do not select an platform listed below.
+      multiple: true
+      options:
+        - linux
+        - windows
+        - macOS
```

Comment:
let's be more specific here and only add the platform's we're currently supporting:
```
- linux/debian
- linux/fedora
- linux/alpine
```